src/Resources/contao/languages/de/tl_style_manager.xlf

```diff
@@ -34,19 +34,23 @@
         <source>New css group</source>
         <target>Neue CSS-Gruppe</target>
       </trans-unit>
-      <trans-unit id="tl_style_manager.editheader.1">
+      <trans-unit id="tl_style_manager.new.1">
+        <source>Create a new css group.</source>
+        <target>Neue CSS-Gruppe erstellen.</target>
+      </trans-unit>
+      <trans-unit id="tl_style_manager.editheader">
         <source>Edit group</source>
         <target>Gruppe bearbeiten</target>
       </trans-unit>
-      <trans-unit id="tl_style_manager.copy.1">
+      <trans-unit id="tl_style_manager.copy">
```

Comment:
I wondered if this might be incompatible with Contao 4.4 - maybe it has to be an array with additional key `0`

Reply:
In fact, the previous state looks a bit incomplete. Even if Contao 4.4 should not be used anymore, I would revert to `0` and `1` here, because otherwise only one letter is output in Contao 4.4.

![image](https://user-images.githubusercontent.com/48379929/114378359-eb917200-9b87-11eb-85d8-b7424e3ddf99.png)

Reply:
That's what I feared